### PR TITLE
fix(executions): Clarify why executions are NOT_STARTED

### PR DESCRIPTION
**Pull request by alanmquach**
Users are sometimes confused why their executions are "stuck" in `NOT_STARTED` state and it's usually because there is a `RUNNING` execution and the pipeline is configured with concurrent executions disabled (the default).

This change just explains it better when `limitConcurrent` is `true`. As a bonus, clicking `RUNNING` also links to executions filtered to running executions of the pipeline in question (there should only be one): 
aquachpy_·_Pipeline_Executions

### CODE REVIEW

**Comment by christopherthielen** [comment]
I ❤️ it.  Travis 🙅‍♀️ doesn't.

**Comment by anotherchrisberry** [comment]
If it were in parentheses I would love it

File: app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx [hunk 1]
```diff
@@ -348,6 +350,19 @@ export class Execution extends React.PureComponent<IExecutionProps, IExecutionSt
               Status:{' '}
               <span className={`status execution-status execution-status-${execution.status.toLowerCase()}`}>
                 {execution.status}
+                {execution.status === 'NOT_STARTED' && execution.limitConcurrent && (
+                  <>
+                    {' '}
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx]:
```suggestion
                    {' ('}
```

File: app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx [hunk 2]
```diff
@@ -348,6 +350,19 @@ export class Execution extends React.PureComponent<IExecutionProps, IExecutionSt
...
+                    waiting on{' '}
+                    <UISref
+                      to={`${$state.current.name.endsWith('.execution') ? '^' : ''}.^.executions`}
+                      params={{ pipeline: execution.name, status: 'RUNNING' }}
+                    >
+                      <a>RUNNING</a>
+                    </UISref>{' '}
+                    executions
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx]:
```suggestion
                    executions)
```

**Comment by alanmquach** [comment]
@anotherchrisberry sounds good.

**Pull request merged**